graviti/manager/commit.py

```diff
@@ -32,20 +52,50 @@ def get(self, revision: Optional[str] = None) -> Commit:
             revision: The information to locate the specific commit, which can be the commit id,
                 the branch name, or the tag name. If it is not given, get the current commit.
 
-        Return:
+        Raises:
+            TypeError: When the given revision is illegal.
+            ResourceNotExistError: When the required commit does not exist.
+
+        Returns:
             The :class:`.Commit` instance with the given revision.
 
         """
+        if revision is None:
+            revision = self._dataset.commit_id
+
+        if not revision:
```

Comment:
In what case would the `commit_id` be empty?

Reply:
if `commit_id=""`, the OpenAPI will return all commits.

Reply:
if `commit_id=""`, the OpenAPI will return all commits.